cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```
@@ -192,6 +192,7 @@ public synchronized void create(final NamespaceMeta metadata) throws Exception {
         masterEnv.onNamespaceCreation(namespace.getNamespace(), metadata.getConfig().getConfigs());
       }
     } catch (Throwable t) {
+      LOG.error(String.format("Failed to create namespace '%s'", namespace.getNamespace()), t);
```

We are also throwing another exception down there. This won't create duplicate log lines, correct?

---

I added this line as I noticed that bootstrap namespace creation errors were being swallowed during retries, so the initial exception gets swallowed. Not sure if it'd be better to throw the error in the bootstrap step or here, but it should be included.

---

Sounds good, just wanted to ensure we don't do double logging